zyte_api/aio/client.py

```diff
@@ -96,11 +96,14 @@ async def request():
                         stats.record_read()
                         stats.record_request_error(content, self.agg_stats)
 
+                        request_id = resp.headers.get("request-id")
+                        message = f"(request_id={request_id}) {resp.reason}"
+
                         raise RequestError(
```

Comment:
What do you think about adding request_id as an attribute to RequestError as well?

Reply:
Good idea. https://github.com/zytedata/python-zyte-api/pull/52/commits/c71fc958367fb4b31aaf87ebbd0ed3ddf9efb56d. Added it in the string representation as well.